enhancements/installer/packet-ipi.md

```diff
@@ -103,22 +131,45 @@ Additional enhancement requests will be created to determine the best approach f
 ### Implementation Details/Notes/Constraints
 
 - RHCOS must be made available and maintained.
-  Images available for provisioning may be custom or official.
-  Official images can be hosted outside of Packet, such as ContainerLinux and RancherOS. More details area available here: <https://github.com/packethost/packet-images>
+
+  Images available for provisioning may be custom or official. Official images
+  can be hosted outside of Packet, such as ContainerLinux and RancherOS. More
+  details area available here: <https://github.com/packethost/packet-images>
   Custom Images can be hosted on git, as described here:
   - <https://www.packet.com/developers/docs/servers/operating-systems/custom-images/>
   - <https://www.packet.com/developers/changelog/custom-image-handling/>
+- It may be possible to boot using the [Packet Custom iPXE
+  configuration](https://www.packet.com/developers/docs/servers/operating-systems/custom-ipxe/).
+
+  As described at <https://ipxe.org/howto/rh_san>, it may be possible to use the
+  [recommended RHCOS bare metal
+  images](https://docs.openshift.com/container-platform/4.5/installing/installing_bare_metal/installing-bare-metal.html#registry-configuring-storage-baremetal_installing-bare-metal)
+  ([mirror](https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.5/?extIdCarryOver=true&sc_cid=701f2000001OH7iAAG)).
+
+  A generic RHCOS Ignition URL could instruct nodes booted on Packet to
+  configure their network using the EC2-like [metadata
```

Comment:
Ignition wouldn't be involved in that process.  Afterburn would automatically run in the initramfs (`ConditionKernelCommandLine=ignition.platform.id=packet`) and configure the network.

Reply:
@liveaverage points out that I should include a note about the size constraints of userdata over Packet metadata.  I'll dig up the limit.

Can Ignition work with base64 encoded, gzipped, multipart mime (like cloud-init can)?

Reply:
No, the userdata has to be an Ignition config without any special encoding.  However, Ignition configs have a mechanism (`ignition.config.merge`) for merging an external Ignition config into the one fetched from metadata.  If necessary, the one in userdata can bootstrap the real config.